sechub-wrapper-prepare/src/main/java/com/mercedesbenz/sechub/wrapper/prepare/modules/WrapperGit.java

```diff
@@ -0,0 +1,82 @@
+package com.mercedesbenz.sechub.wrapper.prepare.modules;
+
+import static com.mercedesbenz.sechub.wrapper.prepare.cli.PrepareWrapperEnvironmentVariables.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import javax.crypto.SealedObject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.mercedesbenz.sechub.commons.core.security.CryptoAccess;
+import com.mercedesbenz.sechub.commons.pds.ProcessAdapter;
+
+@Component
+public class WrapperGit extends WrapperTool {
+
+    private static final Logger LOG = LoggerFactory.getLogger(WrapperGit.class);
+
+    @Autowired
+    JGitAdapter JGitAdapter;
```

Comment:
Maybe rename this field, it first looked like a static call to me.
```suggestion
    JGitAdapter jgitAdapter;
```